### DISCUSSION TITLE
Company Local Landscape, intended use case ?

### DISCUSSION THREAD

**Discussion by senare** [cncf/landscape2#753] [Q&A]
This might be a bit fuzzy, but I was playing around with this as a tool to create an overview/intro/index for what FOSS we make use of at my company.

We are a small company so limited resources which means that realistic alternatives could be something along the line of a wiki page or similar, which for sure would lack the same polished look!

So what I did was pull tooling, and then I cut down the ´https://github.com/cncf/landscape/blob/master/landscape.yml´ to only contain the projects that we make use of, nice !

Update logo, foundation/company information and I can generate a decent looking informative website... so far so good.

Then ofc I would like to add some other FOSS projects to complete the picture ... this very much doable, one concern here is however that I now have a fork from the official data to maintain. 

Additionally, I would like to tone down some parts and extend replace some parts. 

For example, I would like to be able to represent some sort of adoption 'status' internally i.e. we are looking at the project we are using the project we have stopped using the project ... I tried overloading this information in the project maturity (CNCF Sandbox, Incubation Adopted) but this is an ugly hack and kind of interfere with the intended use for these fields.

Or maybe I am more interested in other stats or some subset more relevant for our use case ...

Now I realize I might be kind of outside the scope for the tool, but since it is used to generate both CNCF and some other Landscape sites. I guess the question is ... 

- ...  if this is something that should be possible to support ?
-  ... would it be a wanted use case ? 
-  ... intended ?

Are there other tools or approached that better fit my usecase already ? I don't know of any (suggestions welcome !)

**Comment by tegioz** [comment]
Hi @senare 👋

Like you mentioned, this level of customization is probably out of the scope for this tool. We'd like to keep it as simple as possible, so we focus mainly on features that align well with the landscapes we support.

Maybe you could experiment with a fork and see how it goes for your use case. We know of other users that have created their own to adapt the landscapes generator to suit their needs. If you only need to make some changes here and there, it may not be a problem to maintain the fork going forward 🙂
